docs/OCP/StartingANewProject.md

```diff
@@ -4,52 +4,28 @@ description: The process for getting a new project set up on the BC Gov OpenShif
+The work of different teams within the OpenShift platform is organized into isolated "projects".  Prior to a team beginning work on the platform, they must have a set of projects provisioned for them by the OCIO Enterprise DevOps branch. This document describes the pre-requisites and mechanics of the provisioning process. 
+
+***Each project must be reviewed and accepted, including additional projects for a team that already has project(s) on the platform.*** 
 
 ## Pre-requisites
 
-Prior to project provisioning, a manual/human step is required that involves a project/product team lead / sponsor communicating with the Enterprise DevOps branch leadership and having their project/app/product reviewed and accepted for deployment on the DevOps platform.
+Prior to project provisioning, your team lead/sponsor will need to meet with the Enterprise DevOps branch leadership to have your project/app/product reviewed and accepted for deployment on the DevOps platform.
 
-The projects currently deployed on the platform are largely "Continuous Service Improvement Lab"-affiliated, but there is a set of other well-aligned projects on the platform as well.
+**To request a meeting, please contact [Todd Wilson](mailto:Todd.Wilson@gov.bc.ca)**
 
-Criteria for acceptance onto the platform include:
+### Criteria for acceptance onto the platform:
 
-* executive sponsorship and endorsement of the project and the CSI delivery model
-* based on open source code, with custom code hosted in the BCGov GitHub organization repositories
-* following an agile methodology
-* a sustainable team structure with explicit roles such as DevOps specialist, Scrum Master and Product Owner ideally with one or more of these roles filled by dedicated staff
-* a commitment to participate and contribute in the BC Government open development community  
-
-***Each project must be reviewed and accepted, including additional projects for a team that already has project(s) on the platform.*** 
+* Your executive must sponsor and endorse the project and the continuous service improvement delivery model
+* Your project must be based on open source code, with custom code hosted in the [BCGov GitHub organization repositories](https://github.com/bcgov)
+* Your project must be supported by a team with explicit roles such as DevOps specialist, Scrum Master and Product Owner— ideally with one or more of these roles filled by dedicated staff
+* Your team must be following an agile methodology
+* Your team must have a commitment to participate in and contribute to the BC Government open development community  
 
 # Project Provisioning Request
 
-Once a project is accepted, the team product owner or technical leader should make a request at [DevOps Request Repo](https://github.com/BCDevOps/devops-requests).
```

Comment:
My thinking in removing this was that the DevOps Request Repo now has pre-populated Issue templates that show what info is required, so you don't need to repeat those instructions here anymore. Easier for the user to follow the template while they're in the Issue, rather than think they have to refer back to this page to make sure they haven't missed anything.

Reply:
I think we should still keep this information. Most users start from devhub then they get the instruction to switch to that repo for requests.

Reply:
@mark-a-wilson I think this is good other this. Please pick either me or Shea or Cailey as a reviewer for you PR.

Reply:
@ShellyXueHan - I think the info you want to keep (line 27) is still included - I just reworded it - see the next chunk below ?? Will that work?